src/main/java/chalmers/pimp/model/color/colormodel/IColorModel.java

```diff
@@ -0,0 +1,25 @@
+package chalmers.pimp.model.color.colormodel;
+
+import chalmers.pimp.model.IMementoTarget;
+import chalmers.pimp.model.color.IColor;
+import chalmers.pimp.model.color.colorchangeobserver.IColorChangeObservable;
+
+/**
+ * The {@code IColorModel} holds a color that the model uses.
+ */
```

Comment:
This description isn't that accurate. The interface specifies objects that represent the color model, which has listeners and is observable etc.